packages/models-library/src/models_library/projects_nodes_io.py

```diff
@@ -59,23 +79,17 @@ class BaseFileLink(BaseModel):
     store: Union[str, int] = Field(
         ...,
         description="The store identifier: '0' or 0 for simcore S3, '1' or 1 for datcore",
-        examples=["0", 1],
```

Comment:
this is also something that is nice to have documented. cause we currently allow both int and str

Reply:
i moved the doc to the Configs to have a "complete example"

Reply:
@sanderegg i have a question ... what is the idea behind allowing strings and ints?? Why not simply an enumerator for instance?

Reply:
well... this is maybe the very first field that was ever created... I don't think there is anything behind. I'm happy to set it as whatever you like but then everywhere.